cli/command_repository_set_parameters_test.go

```diff
@@ -39,6 +47,70 @@ func (s *formatSpecificTestSuite) TestRepositorySetParameters(t *testing.T) {
 	require.Contains(t, out, "Max pack length:     44 MiB")
 }
 
+func (s *formatSpecificTestSuite) TestRepositorySetParametersRetention(t *testing.T) {
+	runner := testenv.NewInProcRunner(t)
+	runner.CustomizeApp = func(a *cli.App, kp *kingpin.Application) {
```

Comment:
is there any reason we don't do this always in NewInProcRunner() ?

Reply:
Only question is if we might drop any fs test coverage by doing so. I can also do this in its own PR probably.

Reply:
Ok, I misunderstood, ignore my last comment.

Yes, we can have NewInProcRunner() make the in-memory backend always available for tests.